server/cp/news_event_list.py

```diff
@@ -0,0 +1,43 @@
+STATE_GROUPS = {
+    "atlantic": [
+        "newfoundland and labrador",
+        "nova scotia",
+        "new brunswick",
+        "prince edward island",
+    ],
+    "quebec": ["quebec"],
+    "ottawa": ["ottawa"],
+    "ontario": ["ontario"],
+    "prairies": ["manitoba", "saskatchewan", "alberta"],
+    "british columbia": ["british columbia"],
+    "north": ["nunavut", "northwest territories", "yukon"],
+    "miscellaneous": [],
+}
+
+
+def get_group_items(items, state_group):
+    group_items = []
+
+    for item in items:
+        if item.get("location"):
+            country = item["location"][0]["address"]["country"].lower()
+            locality = item["location"][0]["address"]["locality"].lower()
+
+            if country != "canada" and state_group == "miscellaneous":
+                group_items.append(item)
+            elif country == "canada" and locality in STATE_GROUPS[state_group]:
```

Comment:
How does this go with exclusions (as per the ticket)? i.e. `Ontario` is to exclude `Ottawa` in it's list (i.e. does Events from `Ottawa` appear twice in the exported text?)